data/net/minecraft/client/gui/Font.mapping

```diff
@@ -142,21 +187,61 @@ CLASS net/minecraft/client/gui/Font
 		ARG 2 x
 		ARG 3 y
 		ARG 4 color
-		ARG 5 isShadow
+		ARG 5 drawShadow
 		ARG 6 matrix
 		ARG 7 buffer
-		ARG 8 isTransparent
+		ARG 8 seeThrough
 		ARG 9 colorBackground
 		ARG 10 packedLight
+	METHOD renderText (Lnet/minecraft/util/FormattedCharSequence;FFIZLcom/mojang/math/Matrix4f;Lnet/minecraft/client/renderer/MultiBufferSource;ZII)F
+		ARG 1 text
+		ARG 2 x
+		ARG 3 y
+		ARG 4 color
+		ARG 5 dropShadow
+		ARG 6 matrix
+		ARG 7 buffer
+		ARG 8 seeThrough
+		ARG 9 colorBackground
+		ARG 10 packedLight
+	METHOD split (Lnet/minecraft/network/chat/FormattedText;I)Ljava/util/List;
+		ARG 1 text
+		ARG 2 width
+	METHOD substrByWidth (Lnet/minecraft/network/chat/FormattedText;I)Lnet/minecraft/network/chat/FormattedText;
+		ARG 1 text
+		ARG 2 width
 	METHOD width (Ljava/lang/String;)I
 		COMMENT Returns the width of this string. Equivalent of FontMetrics.stringWidth(String s).
 		ARG 1 text
 	METHOD width (Lnet/minecraft/network/chat/FormattedText;)I
-		ARG 1 properties
+		ARG 1 text
+	METHOD width (Lnet/minecraft/util/FormattedCharSequence;)I
+		ARG 1 text
 	METHOD wordWrapHeight (Ljava/lang/String;I)I
 		COMMENT Returns the height (in pixels) of the given string if it is wordwrapped to the given max width.
-		ARG 1 str
-		ARG 2 maxLength
+		ARG 1 text
+		ARG 2 width
 	CLASS StringRenderOutput
+		METHOD <init> (Lnet/minecraft/client/gui/Font;Lnet/minecraft/client/renderer/MultiBufferSource;FFIZLcom/mojang/math/Matrix4f;Lnet/minecraft/client/gui/Font$DisplayMode;I)V
```

Comment:
This is actually a bit of a weird one, since it is a constructor and most of these parameters therefore are fixed in time. This makes me wonder about the following:
1. Mojang has called the MultiBufferSource parameter 'bufferSource', but I opted for just buffer in the other methods. Should I change it for consistency?
2. Similar with 'dropShadow' vs 'drawShadow' and packedLight' vs 'packedLightCoords' (what Mojang?!)
3. Strangest of all, Mojang named the Matrix4f 'pose', but it is mapped as 'matrix' in most other methods in the class. Should I change that?

Reply:
1. I'd make it `bufferSource`, since the thought is that it's not a buffer itself but merely a source of buffers (and those buffers would be named `buffer`).
2. I'm not too familiar with rendering, but I would lean for `drawShadow` if it's a boolean parameter (possibly `drawDropShadow` if there's the possibility of a non-drop shadow), and `packedLight` (to be more consistent with at least how much I've seen of rendering code).
3. There's the possibility of those other mappings being a holdover from the original MCP (crowdsourced) mappings which we initially based off of. In my own opinion at least (someone with rendering experience will have to comment on this), it would be good to be consistent with Mojang's naming scheme (and also, IIRC the origin of 'pose' is from the `PoseStack`, which is a stack of poses [or matrices]).

Reply:
I eco what Sci said.
``dropShadow`` -> ``drawDropShadow / drawShadow``
``packedLightCoords`` -> ``packedLight``

This is ecoed for both constructors

Reply:
I think I am a bit confused. So you think I should change the parameters to `drawShadow` and `packedLight` even though the fields they are assigned to in the constructor are called `dropShadow` and `packedLightCoords` ?